packages/nocodb/src/models/DataReflection.ts

```
@@ -0,0 +1,177 @@
+      'password',
+      'database',
+    ]);
+
+    const insertedDataReflection = await ncMeta.metaInsert2(
+      RootScopes.ROOT,
+      RootScopes.ROOT,
+      MetaTable.DATA_REFLECTION,
+      insertData,
+    );
+
+    return this.get({ id: insertedDataReflection.id });
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)